.github/workflows/ci.yaml

```diff
@@ -235,6 +235,43 @@ jobs:
           echo "skip_coverage: ${skip_coverage}"
           echo "skip_coverage=${skip_coverage}" >> $GITHUB_OUTPUT
 
+  sync-github-labels:
+    name: Sync GitHub integration labels
+    runs-on: ubuntu-24.04
+    if: github.ref == 'refs/heads/dev'
```

Comment:
Please also make sure that this job isn't run on forks. Either `github.repository_owner == 'home-assistant'` or `!github.event.pull_request.head.repo.fork` should probably work.